core/src/main/scripts/importer/cbioportal_common.py

```diff
@@ -638,90 +639,101 @@ def parse_metadata_file(filename,
                     {True: 'case list', False: 'meta'}[case_list],
                     extra={'filename_': filename,
                            'line_number': line_index + 1})
-                meta_file_type = None
-                return metaDictionary, meta_file_type
+                meta_dictionary['meta_file_type'] = None
+                return meta_dictionary
             key_value = line.split(':', 1)
             if len(key_value) == 2:
-                metaDictionary[key_value[0]] = key_value[1].strip()
+                meta_dictionary[key_value[0]] = key_value[1].strip()
 
+    # Determine meta file type
     if case_list:
         meta_file_type = MetaFileTypes.CASE_LIST
     else:
-        meta_file_type = get_meta_file_type(metaDictionary, logger, filename)
-        # if type could not be inferred, no further validations are possible
-        if meta_file_type is None:
-            return metaDictionary, meta_file_type
+        meta_file_type = get_meta_file_type(meta_dictionary, logger, filename)
+
+    meta_dictionary['meta_file_type'] = meta_file_type
```

Comment:
In contrast to the old code, in the new code `meta_file_type` will be saved in meta_dictionary (`meta_dictionary['meta_file_type'] = meta_file_type`) . I'd like to do this after the if/else, so that it also does this in situation of a case list. Else I have to write the same line twice, in the `if`, and in the `else`. 

This `if meta_file_type is None:` will never be true for 'case lists', because in that situation it just got assigned (in `meta_file_type = MetaFileTypes.CASE_LIST`)